assets/ar/semantics/STS/STSTrack2_BLOOMZ_ZeroShot.py

```diff
@@ -1,14 +1,14 @@
 import os
 import re
 
-from llmebench.datasets import STSArSemEval17Track1Dataset
+from llmebench.datasets import SemEval17T1STSDataset
```

Comment:
Shouldn't this be T2?

Reply:
Yes it should, I think whoever created that asset just used the same dataset loader for both T1 and T2. The data files have the exact same structure so this is why using the same dataset was fine.

Reply:
Done in e812c9d